pkg/discovery/dtofactory/pod_entity_dto_builder.go

```diff
@@ -318,10 +318,12 @@ func (builder *podEntityDTOBuilder) createContainerPodData(pod *api.Pod) *proto.
 	// in which case the IP address will not be unique (in the k8s cluster) and hence not populated in ContainerPodData.
 	fullName := pod.Name
 	ns := pod.Namespace
+	port := "na"
```

Comment:
One pod may have multiple containers and each container may or may not have ports. So, it is not straightforward to set the port. Currently, we don't use this data anywhere and this setting is to avoid error at server side when executing action. So, I am not setting this info. Will use more expressive string for it.